src/CUPROBE.CPP

```diff
@@ -845,7 +845,8 @@ breakBreak:		// show j input members, k run members
 LOCAL void FC NEAR disMember( SFIR *f1, SI isIn, SI isRun, SI showAll)	// display info on one record member, for shoProbeNames
 {
 	USI dt, ty;
-	char *tySubTx="?", *evfSubTx;
+	char *tySubTx="?";
+	const char *evfSubTx;
```

Comment:
Maybe since you're fixing the type anyway, just move the declaration to where the assignment already is.

Reply:
Okay.